src/musig.h

```diff
@@ -12,6 +12,10 @@
 
 struct secp256k1_musig_keyagg_cache;
 
+//! MuSig2 chaincode as defined by BIP 328
+//! uint256 will byteswap the hex
+constexpr uint256 MUSIG_CHAINCODE{"6589e367712c6200e367717145cb322d76576bc3248959c474f9a602ca878086"};
```

Comment:
in commit 048c95ea557cf81a8e55c5ffe9994cc9d8f3e039: not too fond of "reverse-hexstrings of byte-strings"  creeping in, could do instead
```suggestion
using namespace util::hex_literals;
constexpr std::array<uint8_t, 32> MUSIG_CHAINCODE{"868087ca02a6f974c4598924c36b57762d32cb45717167e300622c7167e38965"_hex_u8};
```
and assign the constant to the uint256 instance via `... = uint256(MUSIG_CHAINCODE)`.
(but shouldn't be a blocker, can be improved in a follow-up as well, if others are even as annoyed by that as me 😅 )

Reply:
Indeed it's nice if we can avoid this, because although reviewing the reversal was easy, finding it later with search is tedious.

Reply:
It looks like `_hex_u8` can be used directly in uin256's constructor with normal byte order, so used that.